timewarp_gl/plugin.cpp

```diff
@@ -64,7 +65,27 @@ class timewarp_gl : public threadloop {
 		  // In production systems, this is certainly a good thing, but it makes the system harder to analyze.
 		, disable_warp{ILLIXR::str_to_bool(ILLIXR::getenv_or("ILLIXR_TIMEWARP_DISABLE", "False"))}
 		, enable_offload{ILLIXR::str_to_bool(ILLIXR::getenv_or("ILLIXR_OFFLOAD_ENABLE", "False"))}
-	{ }
+	{ 
+		image_handles_ready = false;
+		swapchain_ready = false;
+		sb->schedule<image_handle>(id, "image_handle", [this](switchboard::ptr<const image_handle> handle, std::size_t) {
+			if (handle->swapchain_index > 1) {
+				return;
+			}
+			
+			assert(handle->swapchain_index == 0 || handle->swapchain_index == 1);
```

Comment:
More than two swapchains are actually possible. For example, Varjo headsets have two displays per eye -- an inner one and an outer one -- and each gets its own swapchain. Therefore, there are a total of four swapchains. Of course, supporting such a display would probably require some other changes too, so leave a comment here saying we only support two swapchains for now.